lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -307,6 +360,24 @@ def set_fn(value):
             self._fig = plt.figure()
             self._axes = self._fig.add_subplot(111)
 
+    def _select_state(self, value):
+        """Select a different state to operate the widget in. Note that the input argument is value
+        because it is hooked up to a ToggleButton directly"""
+        self._area_selector.set_active(False)
+        self._line_connector.set_active(False)
+
+        if value["new"] == "Track lines":
+            self._set_label(f"Drag right mouse button to track lines")
+            self._area_selector.set_active(True)
+            self.adding = True
+        elif value["new"] == "Remove lines":
+            self._set_label(f"Drag right mouse button to remove lines")
+            self._area_selector.set_active(True)
+            self.adding = False
+        elif value["new"] == "Connect lines":
+            self._set_label(f"Drag right mouse button to connect lines")
+            self._line_connector.set_active(True)
```

Comment:
`self.adding[False]`
Or is this not necessary here?

Reply:
It's not required here, since that argument is only used by the trace addition / removal code.